probe-rs/src/bin/probe-rs/cmd/dap_server/debug_adapter/dap/adapter.rs

```diff
@@ -1101,7 +1101,7 @@ impl<P: ProtocolAdapter> DebugAdapter<P> {
                 let function_display_name = if frame.is_inlined {
                     format!("{} #[inline]", frame.function_name)
                 } else {
-                    format!("{} @{}", frame.function_name, frame.pc)
+                    frame.function_name.clone()
```

Comment:
This clone should not be required because frame will be thrown away afterwards.

Reply:
The compiler disagrees. Am I missing something?

Reply:
I am not sure what would need to be changed - I just have a hunch that it is superfluous in the grand scheme of things :)

Reply:
The `function_display_nanme` is a `String` and here I am assigning the `function_name` field of `frame`, which is a reference ... I don't think `clone()` or some other new `String` creating can be avoided. The compiler requires it, so until I learn of a better way ... :)